src/implementations/vanilla/components/carousel/_carousel.scss

```diff
@@ -85,71 +85,78 @@ $_outline-width: null !default;
 
 .ecl-carousel__prev,
 .ecl-carousel__next {
-  background: $_button-arrows-background;
+  background: $_controls-background;
   border: 0;
   cursor: pointer;
-  display: none;
-  padding: map.get(theme.$spacing, 's');
-  position: absolute;
-  top: calc(50% - 22px);
-  transform: translateY(-50%);
+  display: block;
+  padding: map.get(theme.$spacing, 'xs');
 
-  &:focus {
-    outline: $_outline-width solid $_outline-color;
-  }
-}
+  &:active,
+  &:hover {
+    background-color: map.get(theme.$color, 'white-100');
 
-.ecl-carousel__prev {
-  left: 0;
+    .ecl-carousel__icon-default {
+      fill: $_controls-background;
+    }
+  }
 
   &:focus {
-    margin-inline-start: $_outline-width;
-    padding-inline-start: calc(
-      #{map.get(theme.$spacing, 's')} - #{$_outline-width}
-    );
+    outline: $_outline-width solid $_outline-color;
   }
 }
 
-.ecl-carousel__next {
-  right: 0;
-
-  &:focus {
-    margin-inline-end: $_outline-width;
-    padding-inline-end: calc(
-      #{map.get(theme.$spacing, 's')} - #{$_outline-width}
-    );
-  }
+.ecl-carousel__autoplay {
+  display: none;
```

Comment:
the buttons don't disapear at the the same breakpoint as the mobile version, so there are some cases where you have the mobile display, but still have the button

Reply:
Yes that's correct, https://www.figma.com/file/wAuZXP2e9NUrsT5C1EKNEk/Carousel?node-id=428%3A32040